traiderdaive/utils/set_seeds.py

```diff
@@ -0,0 +1,12 @@
+import random
+
+import numpy as np
+import torch
+
+
+def set_seeds(seed: int | None = None):
+    seed = seed if seed is not None else np.random.randint(1, 99999)
+    random.seed(seed)
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+    print(f"Global Seed: {seed}")
```

Comment:
I think we use logging elsewhere, might as well do it here.

Reply:
haha, I initially was using logging, but ray internally sets up the logger during config.build(), so if I call set_seeds before config.build(), it doesn't get outputted, and if I call set_seeds after that, the seed doesn't actually work.